djaopsp/management/commands/completed_assessments.py

```diff
@@ -26,9 +26,9 @@ def add_arguments(self, parser):
                             action='store',
                             help='Slug of the campaign',
                             default=None)
-        parser.add_argument('--grantee',
-                            action='store',
-                            help='Slug of grantee')
+        parser.add_argument('--grantees',
+                            nargs='+',
```

Comment:
That's interesting. As a convention so far we had used repetition of the command line parameter, i.e. `python manage.py command --grantee grantee1 --grantee grantee2 ...`. Here it seems the convention is `python manage.py command --grantees grantee1,grantee2 ...`